### ISSUE TITLE
Franco Kernel Manager Removed from Playstore?

### ISSUE THREAD

**Issue by sonhak-18** [franciscofranco/FK-Kernel-Manager-IssuesAndFeatureRequest#170]
Very surprising. I can't find FKM in play store. Has anyone experienced this too?

**Comment by SnOOpeRa** [comment]
yep.. and the app stop working on 3 devices ( already installed on them for years from play store - paid and free one ) , as of seems play store check on load happen and black screen for piracy or so overlay popup and to download it from play store button... how to bypass this screen ?

**Comment by kshitij-vk** [comment]
> Very surprising. I can't find FKM in play store. Has anyone experienced this too?

This is bad . People with pirated copies are flexing to me and I can't install it 😂😂😂😂

**Comment by jsjajajjajaashhbsbabhsshhshs** [comment]
What happened? This is already happened few times, I bought the app on playstore but it says " This doesn't appear to be licensed " like what the hell?
I think i bought the wrong kernel manager this app sucks

**Comment by Sadyantssr** [comment]
Same thing here. Can't use app to run Scripts now. Any links for pirated version?

**Comment by minhgi** [comment]
Yep... Same issue too. What happened to the app from the play store.

**Comment by Lanquiv** [comment]
I can't use the app because it's gone from google play :(

**Comment by ZenkaBestia** [comment]
@franciscofranco it's the simplest yet complete functionality app and still no other app can replace it, please do some update.

**Comment by kshitij-vk** [comment]
If anyone still has clean, non cracked apk. Please do share it.

**Comment by ZenkaBestia** [comment]
> If anyone still has clean, non cracked apk. Please do share it.

what's you gonna do..? it can't be used anyway..

**Comment by marcoh242** [comment]
Removing License Verification with LuckyPatcher works.
